packages/core/src/__tests__/logger.ts

```diff
@@ -0,0 +1,7 @@
+export const logger = {
```

Comment:
Don't we want to reference the `Logger` interface here? Otherwise TS won't help us, when we should decide to change the interface.